### ISSUE TITLE
[Bash 5.1, bash-completion] bash: _minimal '': command not found

### ISSUE THREAD

**Issue by huresche** [akinomyoga/ble.sh#116]
**ble version**: 0.4.0-devel3+c94d292
**Bash version**: 5.1.8(1)-release (x86_64-pc-linux-gnu)

Double quoted special parameters give an error when they are typed without a preceding argument, including inside functions. This also happens with a `\` followed by a character inside double quotes with no preceding argument. The error message:
```
bash: _minimal '': command not found
bash: _minimal '': command not found
```
The combinations:

`"\a"` → error
`'\a'` → OK
`echo "\a"` → OK
`"$@"` → error
`"$*"` → error
`"$#"` → error
`"$$"` → error
`"$-"` → error
`"$0"` → OK
`"$1"` → error
`"$2"` → error
`"$3"` → OK
`"$_"` → OK
`a(){ "$@";}` → error
`a(){ echo "$@";}` → OK

This error doesn't show after running `complete -r`

**Comment by akinomyoga** [comment]
Thank you for the report! Fixed it 301d40f. Could you check if the problem has been solved?

**Comment by huresche** [comment]
The problem is solved!
